lms/templates/shoppingcart/shopping_cart.html

```diff
@@ -9,140 +9,143 @@
-
-           <img style="width: 100%; height: 100%;" src="${course_image_url(course)}"
-                alt="${course.display_number_with_default | h} ${get_course_about_section(course, 'title')} Cover Image" />
-
-      </div>
-      <div class="data-input">
-        <h3>${_('Registration for:')} <span class="pull-right">${_('Course Dates:')}</span></h3>
-        <h1>${ course.display_name }</h1><span class="pull-right">${course.start_datetime_text()} -  ${course.end_datetime_text()}</span>
-        <hr />
-        <div class="three-col">
-            <div class="col-1">
-                % if item.list_price != None:
-                <% discount_applied = True %>
-                <div class="price">${_('Price per student:')} <span class="line-through">  ${currency_symbol}${"{0:0.2f}".format(item.list_price)}</span></div>
-                <div class="price green">${_('Discount Applied:')} <span>  ${currency_symbol}${"{0:0.2f}".format(item.unit_cost)} </span></div>
-                % else:
-                <div class="price">${_('Price per student:')} <span>  ${currency_symbol}${"{0:0.2f}".format(item.unit_cost)}</span></div>
-                % endif
-            </div>
-            <div class="col-2">
-              <div class="numbers-row">
-                <label for="students">${_('Students:')}</label>
-                <div class="counter">
-                    <input maxlength="3" title="Input qty and press enter." max="999" type="text" name="students" value="${item.qty}" id="${item.id}" data-unit-cost="${item.unit_cost}" data-qty="${item.qty}">
+        </div>
+        <div class="data-input">
+          <h3>${_('Registration for:')} <span class="pull-right">${_('Course Dates:')}</span></h3>
+          <h1>${ course.display_name }</h1><span class="pull-right">${course.start_datetime_text()} -  ${course.end_datetime_text()}</span>
+          <hr>
+          <div class="three-col">
+              <div class="col-1">
+                  % if item.list_price != None:
+                    <% discount_applied = True %>
+                    <div class="price">${_('Price per student:')} <span class="line-through">  ${currency_symbol}${"{0:0.2f}".format(item.list_price)}</span></div>
+                    <div class="price green">${_('Discount Applied:')} <span>  ${currency_symbol}${"{0:0.2f}".format(item.unit_cost)} </span></div>
+                  % else:
+                    <div class="price">${_('Price per student:')} <span>  ${currency_symbol}${"{0:0.2f}".format(item.unit_cost)}</span></div>
+                  % endif
+              </div>
+              <div class="col-2">
+                <div class="numbers-row">
+                  <label for="students">${_('Students:')}</label>
+                  <div class="counter">
+                      <input maxlength="3" title="${_('Input qty and press enter.')}" max="999" type="text" name="students" value="${item.qty}" id="${item.id}" data-unit-cost="${item.unit_cost}" data-qty="${item.qty}">
```

Comment:
Can you use "quantity" instead of "qty"? If not, please add a translators comment explaining that "qty" is short for "quantity". http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#editing-source-files